kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/converter/altoida/summary/AltoidaTestMetricsProcessor.kt

```diff
@@ -18,201 +18,201 @@ class AltoidaTestMetricsProcessor(private val type: AltoidaTestCategory, val top
-            "${type.prefix}NORMCIRCLESPEEDACCURACYRATIO",
-            "${type.prefix}NORMCIRCLEDURATION",
-            "${type.prefix}NORMCIRCLEFULLDURATION",
-            "${type.prefix}NORMSQUAREMEAN",
-            "${type.prefix}NORMSQUAREVARIANCE",
-            "${type.prefix}NORMSQUARETIMERATIOWITHIN",
-            "${type.prefix}NORMSQUAREDISTRATIOWITHIN",
-            "${type.prefix}NORMSQUAREDISTWITHIN",
-            "${type.prefix}NORMSQUAREDISTTOTAL",
-            "${type.prefix}NORMSQUAREANGLESUM",
-            "${type.prefix}NORMSQUAREDIRECTIONALCHANGES",
-            "${type.prefix}NORMSQUARESPEED",
-            "${type.prefix}NORMSQUARESPEEDACCURACYRATIO",
-            "${type.prefix}NORMSQUAREDURATION",
-            "${type.prefix}NORMSQUAREFULLDURATION",
-            "${type.prefix}NORMSERPENTINEMEAN",
-            "${type.prefix}NORMSERPENTINEVARIANCE",
-            "${type.prefix}NORMSERPENTINETIMERATIOWITHIN",
-            "${type.prefix}NORMSERPENTINEDISTRATIOWITHIN",
-            "${type.prefix}NORMSERPENTINEDISTWITHIN",
-            "${type.prefix}NORMSERPENTINEDISTTOTAL",
-            "${type.prefix}NORMSERPENTINEYPROGRESS",
-            "${type.prefix}NORMSERPENTINESPEED",
-            "${type.prefix}NORMSERPENTINESPEEDACCURACYRATIO",
-            "${type.prefix}NORMSERPENTINEDURATION",
-            "${type.prefix}NORMSERPENTINEFULLDURATION",
-            "${type.prefix}NORMSPEEDCIRCLEMEAN",
-            "${type.prefix}NORMSPEEDCIRCLEVARIANCE",
-            "${type.prefix}NORMSPEEDCIRCLETIMERATIOWITHIN",
-            "${type.prefix}NORMSPEEDCIRCLEDISTRATIOWITHIN",
-            "${type.prefix}NORMSPEEDCIRCLEDISTWITHIN",
-            "${type.prefix}NORMSPEEDCIRCLEDISTTOTAL",
-            "${type.prefix}NORMSPEEDCIRCLEANGLESUM",
-            "${type.prefix}NORMSPEEDCIRCLEDIRECTIONALCHANGES",
-            "${type.prefix}NORMSPEEDCIRCLESPEED",
-            "${type.prefix}NORMSPEEDCIRCLESPEEDACCURACYRATIO",
-            "${type.prefix}NORMSPEEDCIRCLEDURATION",
-            "${type.prefix}NORMSPEEDCIRCLEFULLDURATION",
-            "${type.prefix}NORMRANDOMTAPPINGMEAN",
-            "${type.prefix}NORMRANDOMTAPPINGVARIANCE",
-            "${type.prefix}NORMRANDOMTAPPINGCOUNT",
-            "${type.prefix}NORMRANDOMTAPPINGREACTIONTIMES",
-            "${type.prefix}NORMRANDOMTAPPINGDURATION",
-            "${type.prefix}NORMRANDOMTAPPINGRATE",
-            "${type.prefix}NORMRANDOMTAPPINGFULLDURATION",
-            "${type.prefix}NORMTAPPINGMEAN",
-            "${type.prefix}NORMTAPPINGVARIANCE",
-            "${type.prefix}NORMTAPPINGCOUNT",
-            "${type.prefix}NORMTAPPINGREACTIONTIMES",
-            "${type.prefix}NORMTAPPINGRATE",
-            "${type.prefix}NORMTAPPINGDURATION",
-            "${type.prefix}NORMTAPPINGFULLDURATION")
+            "${type.prefix}AR_HIGHTONEREACTIONTIMES",
+            "${type.prefix}AR_HIGHTONETOUCHACCURACY",
+            "${type.prefix}AR_NMRREACTIONSTOLOWTONE",
```

Comment:
@nivemaham `BIT_AR_NMRREACTIONSTOLOWTONE` is in the excel mapping file but not in the `export.csv` (although it is present for the `DOT_` metrics). Should this always be there or should we account for the instances where it's not present?

Reply:
Lets make it nullable in schema and add it?

Reply:
Okay sure. But when it comes to matching the headers, the column is still there so it doesn't go through the `AltoidaTestMetrics` processor, is it fine if I remove it from there? Also, what if there are other cases like this for other columns?

Reply:
Never mind, it seems only `BIT_AR_NMRREACTIONSTOLOWTONE` is missing.